package-res/resources/web/pentaho/data/filter/AbstractFilter.js

```diff
@@ -35,6 +32,8 @@ define([
    * @classdesc The `AbstractFilter` class is the abstract base class of
    * classes that represent a filter.
    *
+   * @description The `AbstractFilter` interface allows creation of types of filters for a data table.
+   *
```

Comment:
Here would be a good place to explain the idea behind filters, and that we are using them to express a subset of data in an intensional manner.